# Used for distributing distance sensor readings

# Standard ROS message header
Header header
# An array of DistanceValue messages represent include the sensor readings
DistanceValue[] values
